connectors/salesforce/src/test/java/com/vmware/connectors/salesforce/SalesforceControllerTest.java

```diff
@@ -389,7 +441,9 @@ private void testRequestCards(String requestFile, String responseFile, String ac
                 .getResponseBody()
                 .collect(Collectors.joining())
                 .map(JsonNormalizer::forCards)
+                .map(json -> json.replaceAll("http://localhost:\\d+/", "https://salesforce.acme.com/"))
                 .block();
+        System.out.println(body);
```

Comment:
Don't forget to take this out.

Reply:
My bad. I have kept this for debugging and I missed to remove it.